index.html

```diff
@@ -175,8 +175,30 @@ <h1 id="hire-me-h1">I am available for freelance work.</h1>
     </div>
     <!-- our work -->
     <div class="container">
-        <div class="row">
-            OUR WORK GALLERY
+        <div class="row our-work-section">
+            <div class="our-work-header">
+                <h2>Our <strong>Work</strong></h2>
+                <div class="flex-parent snowflake">
+                    <div class="flex-child-edge"></div>
+                    <div class="flex-child-text">&#42;</div>
+                    <div class="flex-child-edge"></div>
+                </div>
+                <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at this layout.</p>
+                <div class="filter-box">
+                    <ul class="filter-ul">
+                        <li class="filter-list-item"><a href="">All</a></li>
+                        <li class="star">&#42;</li>
```

Comment:
Tokias žvaigždutes realizuoti panaudojant pseudo elementus (pvz ::before)

Reply:
Changed star symbol. 
Need to update bulletpoints -- > element name has to be updated in header / or class added. (ul / li), cause my section inherits everything by default